### ISSUE TITLE
BOOM and Allegro freeze

### ISSUE THREAD

**Issue by ezgoodnight** [MiSTer-devel/ao486_MiSTer#128]
Tested in working DOS 6.22 and FreeDOS environments. All versions of BOOM and Allegro (ASETUP, the setup EXE) freeze when I try to run them. It locks at a blinking cursor, I've waited for 10 minutes or so.

(DOOM and DOOM2 all work in FreeDOS just fine.)

This is the version that comes packaged with other versions of FreeDOS, plus I have tried the classic version of BOOM available online.

**Comment by ezgoodnight** [comment]
NobleEightfold in the discord suggested I [try updating the BOOM.exe with DOS32A as per this page](https://www.dosbox.com/wiki/TOOLS:DOS32A).

I confirmed that this works with DOOM but not BOOM, so I assume I did it right testing it for both EXE files.
![sshot-1167](https://user-images.githubusercontent.com/11281259/226109111-cfb3392b-05bf-47ae-a185-67b495440218.png)
The error code says "not supported" or not compiled correctly. The [list of error codes here](https://www.javiergutierrezchamorro.com/wp-content/uploads/2013/07/dos32a/html/user/8.html) also says the exe might be corrupt but I do not think that is the case as it came from the same download where it works in PCem and DOSbox.
